### ISSUE TITLE
--config file.json - paths in file.json are relative to current execution path, not relative to location of file.j

### ISSUE THREAD

**Issue by the-docta** [SAP/cloud-sdk#633]
**Describe the bug**
The output of `--help` clearly states that the paths inside the config file are relative to the folder where the config file is stored:

```
  -c, --config                       Instead of specifying the options on the
                                     command line, you can also provide a path
                                     to single .json file holding these options.
                                     The file must be a valid .json file where
                                     the keys correspond to the command line
                                     flags without dashes. Paths will be
                                     interpreted relative to the config file.
```
now, if I place the file 
```
{
    "inputDir": "",
    "outputDir": "client",
    "clearOutputDir": true
}
```
in the folder `srv/external` and run `generate-odata-cloent -c srv/external/file.json` inside the folder containing srv, it tries to find metadata files ewerywhere in the folder containing src.

If, OTOH, the file contains this:
```
{
    "inputDir": "srv/external",
    "outputDir": "srv/external/client",
    "clearOutputDir": true
}
```
run `generate-odata-cloent -c srv/external/file.json`, then client generator finds the metadata files contained in srv/external.

**To Reproduce**
Steps to reproduce the behavior: see above

**Expected behavior**
Either the --help output should be changed to say "paths are interpreted relative to the path where generate-odata-client is executed" or the behavouur should be changed to the one described in --help

**Used Versions:**
 - node version via `node -v`
v12.19.0
 - npm version via `npm -v`
6.14.8
- `generate-odata-client --version   `
1.30.0

**Please check the following points**
- [x] Have you provided detailed description about your issue?
- [x] Have you shared all the necessary information to help the cloud-sdk developer reproduce the issue?
- [x] Have you provided the expected behaviour?
- [-] Have you attached some useful screenshots?
- [x] Have you provided the node/npm/cloud-sdk versions?
- [x] Have you shared your code snippets as a minial example?
- [-] Have you provided you log file or related error messages?
- [-] Have you searched for related issues on [Stackoverflow](https://stackoverflow.com/questions/tagged/sap-cloud-sdk) or [answers.sap.com](https://answers.sap.com/tags/73555000100800000895)?

**Comment by artemkovalyov** [comment]
Hi @the-docta ,

We'll take a look at this shortly. Thanks for sharing the reproduction steps.

Do I understand correctly it's not blocking as you have a successful execution path and this is an inconsistency between the help output and the actual way to resolve input files for the generator?
cc @marikaner 

Best,
Artem

**Comment by the-docta** [comment]
> Do I understand correctly it's not blocking as you have a successful execution path and this is an inconsistency between the help output and the actual way to resolve input files for the generator?

precisely.

Thanks!

**Comment by marikaner** [comment]
Hey @the-docta,

good catch, thanks for reporting this. I created a pull request to fix this. I think the documented behaviour is the favorable one.

**Comment by marikaner** [comment]
Hey @the-docta,

The bug has been fixed in version 1.31.0. Let us know if you still encounter any issues with this.
